draft-packages/avatar/KaizenDraft/Avatar/AvatarGroup.tsx

```diff
@@ -0,0 +1,84 @@
+import * as React from "react"
+import cx from "classnames"
+import { v4 } from "uuid"
+import { Avatar, GenericAvatarProps, CompanyAvatarProps } from "./Avatar"
+import styles from "./AvatarGroup.module.scss"
+
+export type AvatarProps =
+  | Omit<GenericAvatarProps, "size">
+  | Omit<CompanyAvatarProps, "size">
+
+export type AvatarGroupSize = "small" | "medium" | "large"
+export type AvatarList = [AvatarProps, ...AvatarProps[]]
```

Comment:
Curious about this type, what does this do different from `AvatarProps[]`?

Reply:
This is a way to guard against a non empty array - it'll throw a ts error if an empty array is provided, where as AvatarProps[] won't. Not sure if it's overkill but good to know it exists

Reply:
Oh nice, I learned something :)